paper/paper.md

```diff
@@ -0,0 +1,203 @@
+The full optical response of a multilayer system is then determined by matrix multiplication of the layers' matrices.
+
+The ellipsometer hardware typically supports bundled proprietary software solutions for such analyses.
+Unfortunately, each manufacturer supplies their own adapted tools.
+This approach promises efficient laboratory workflows if working flawlessly.
+However, it binds scientists to specific optical models and experiments available in the provided software.
+Furthermore, data interchange can be cumbersome, and results may be hard to reproduce on competitive systems due to the use of other models or parameters.
+In addition, limitations of the bundled software may stimulate scientists to use third-party software: bundled software packages may not support specific kinds of analyses, such as including the response of optically anisotropic materials or simultaneous fitting of external experimental parameters.
+
+**PyElli** offers an open-source alternative extending the capabilities of existing solutions, while aiming to remain as compatible as possible, by providing data imports from various manufacturers (Woollam VWASE, Woollam CompleteEASE, Sentech, Accurion).
+The code is designed with extensibility and adaptability in mind enabling the implementation of individually adapted models as well as data evaluation with custom tools.
+Typical examples for advanced use-cases are implementations of custom experimental geometries not covered by other software [@eberheim2022], imaging ellipsometry, or as a full FAIR data automated analysis pipeline for SE measurements.
+**PyElli** also supports recent advances in the standardization of ellipsometry data and models, addressing the need for FAIR data [@Wilkinson2016].
+
+**PyElli** aims to provide a straight-forward database of predefined dispersion models for analyzing materials.
+All optical models adhere closely to the literature [@Hilfiker2018].
+The software easily includes the popular public-domain database for optical constants [refractiveindex.info](https://refractiveindex.info) [@rii].
+This allows the inclusion of literature dispersions with a single line of code.
+Additional dispersion relations can be either hard coded, which is more efficient, or parsed from a text-based domain-specific language into a fittable dispersion.
+
+**PyElli** supports multiple solving algorithms with different characteristics.
+Currently, two algorithms using different formulations are available: a fast algorithm based on a 2x2 matrix formulation [@byrnes2020multilayer] and a more complex 4x4 matrix formulation [@Berreman72; @berreman4x4_doku; @berreman4x4_software].
+The 2x2 matrix algorithm divides the light into two perpendicular linearly polarized beams, which are solved separately.
+One of its limitations is eliminating the possibility to include birefringent materials.
+The 4x4 matrix approach fully solves Maxwell's equations.
```

Comment:
I wouldn't say it solves them fully. It solves them fully in the frame of a 1D model. For modelling 2D or 3D interaction one would need to use RCWA or other algorithms to solve this.